src/words/TokTok.jl

```diff
@@ -110,19 +112,21 @@ function toktok_tokenize(instring::AbstractString)
             flush!(ts, ts.input[i:end])
         end
     end
- 
+    
     while !isdone(ts) && ts.idx <= flag
-       atoms(ts, vcat(rules_atoms...)) || 
+      atoms(ts, Tuple(Iterators.flatten(rules_atoms))) || 
        replaces(ts, vcat(rules_replaces...)) || 
        replaces(ts, vcat(LSTRIP, RSTRIP), boundary = true) ||
```

Comment:
similar for these `vcat`s.
No need to do all that work every loop iteration when the result is constant.